packages/web-components/docs/api-report.md

```diff
@@ -1248,7 +1247,7 @@ export const ProgressBarValidationState: {
 export type ProgressBarValidationState = ValuesOf<typeof ProgressBarValidationState>;
 
 // @public
-export const setTheme: (theme: Theme) => void;
+export const setTheme: (theme: any) => void;

```

Comment:
I don't think we want to remove this, I think the PR Martin has in will fix this and address the problem.